src/librustc_incremental/assert_dep_graph.rs

```diff
@@ -272,10 +267,14 @@ impl<'a, 'tcx, 'q> dot::Labeller<'a> for GraphvizDepGraph<'q> {
         dot::Id::new("DependencyGraph").unwrap()
     }
     fn node_id(&self, n: &&'q DepNode<DefId>) -> dot::Id {
-        let s: String =
-            format!("{:?}", n).chars()
-                              .map(|c| if c == '_' || c.is_alphanumeric() { c } else { '_' })
-                              .collect();
+        let s: String = format!("{:?}", n)
+            .chars()
+            .map(|c| if c == '_' || c.is_alphanumeric() {
+                c
+            } else {
+                '_'
+            })
+            .collect();
```

Comment:
Yeah, this is probably the trickiest corner of formatting at the moment - I prefer visual indent for method chains, but it makes closures either ugly or subject to extreme rightward drift.